### PR TITLE
fix: fix SquareLayout's outline caculation

### PR DESCRIPTION
**Pull request by daybrush**
## Issue

#336

### CODE REVIEW

File: src/layouts/SquareLayout.ts [hunk 1]
```diff
@@ -82,7 +82,7 @@ export default class SquareLayout extends FrameLayout {
 		const columnLength = this.options.column ||
 			Math.floor((this._size + margin) / (itemSize + margin)) || 1;
 		const length = items.length;
-		const endOutline = makeShapeOutline(outline, itemSize, columnLength, isAppend);
+		const endOutline = makeShapeOutline(outline, Math.floor(itemSize), columnLength, isAppend);
```
Comment (mixed) [inline src/layouts/SquareLayout.ts:85]: Is there a similar issue with other layouts?

Comment (daybrush) [reply src/layouts/SquareLayout.ts:85]: @mixed

yes. Only SquareLayout happens.

File: test/unit/layouts/SqaureLayout.spec.js [hunk 2]
```diff
@@ -217,19 +217,26 @@ describe("SquareLayout Test", function () {
...
+				// 0 1 2 3 4 (5)
+				// 6 7 8 9 X  X
+				const group2 = layout.append(items.slice(5, 10), group1.outlines.end);
 
 				// Then
-				expectOutlineIndex(layout, group);
-				expectOutlineIndex(layout, group2);
+				expect(group2.items[0].rect.left).to.be.not.equal(0);
 			});
+			it
```
Comment (WoodNeck) [inline test/unit/layouts/SqaureLayout.spec.js:239]: 'it' is unremoved.
And, sorry for the late review.
Please tell me to review PR before merging, sometimes I forgot to do so when I'm working on other projects.

Comment (daybrush) [reply test/unit/layouts/SqaureLayout.spec.js:239]: @WoodNeck 

Ok. Thank you.

**Pull request merged**